assets/src/edit-story/components/carousel/zoomSelector/zoomSelector.js

```diff
@@ -43,32 +43,52 @@ const StyledDropDown = styled(DropDown)`
 const ZOOM_OPTIONS = [
   {
     label: _x('100%', 'zoom level', 'web-stories'),
-    value: ZOOM_SETTING.SINGLE,
+    value: 1,
   },
   {
     label: _x('200%', 'zoom level', 'web-stories'),
-    value: ZOOM_SETTING.DOUBLE,
+    value: 2,
   },
   { label: _x('Fill', 'zoom level', 'web-stories'), value: ZOOM_SETTING.FILL },
   { label: _x('Fit', 'zoom level', 'web-stories'), value: ZOOM_SETTING.FIT },
 ];
 
 function ZoomSelector() {
-  const { zoomSetting, setZoomSetting } = useLayout(
-    ({ state: { zoomSetting }, actions: { setZoomSetting } }) => ({
+  const { zoomSetting, zoomLevel, setZoomSetting, setZoomLevel } = useLayout(
+    ({
+      state: { zoomSetting, zoomLevel },
+      actions: { setZoomSetting, setZoomLevel },
+    }) => ({
       zoomSetting,
+      zoomLevel,
       setZoomSetting,
+      setZoomLevel,
     })
   );
 
-  const placeholder = useMemo(
-    () => ZOOM_OPTIONS.find(({ value }) => value === zoomSetting).label,
-    [zoomSetting]
-  );
+  const placeholder = useMemo(() => {
+    const option = ZOOM_OPTIONS.find(({ value }) => {
+      if (zoomSetting === ZOOM_SETTING.FIXED) {
+        return value === zoomLevel;
+      }
```

Comment:
Is this even necessary? Would the [default handling in line 79](https://github.com/google/web-stories-wp/pull/7569/files#diff-ce0292d29b7dee1c79ee6aad7ccbdf554bcae990a901e15eac52dfd161acbc3bR79) handle this too? Unless someone of course translates "100%" to something else - like "Normal size".